react/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "redux-loop/react",
+  "description": "React specific declarations for redux-loop",
```

Comment:
we shouldn;t have all of these fields in this file. we really only need name and types. and probably set private to true. we don't want to be publishing this on its own.